src/com/dakl/Main.java

```diff
@@ -61,6 +64,7 @@ public static void main(String[] args) {
         System.out.println(output);
     }
 
+    // REVIEW : javadocs to explain "why" 
```

Comment:
This is a high-level comment about having javadocs on methods that are not trivial that explain the purpose / "the why" of the method. You don't have to go overboard in describing the "why" of getters and setters, but more complex methods with more than one parameters often benefit from explaining what it does (otherwise, I have to read a bunch of complicated logic to figure it out).